src/jit/codegenlegacy.cpp

```diff
@@ -8620,11 +8638,13 @@ void                CodeGen::genCodeForShift(GenTreePtr tree,
         reg = op1->gtRegNum;
 
         /* Are we shifting left by 1 bit? (or 2 bits for fast code) */
+        CLANG_FORMAT_COMMENT_ANCHOR;
+
+#ifndef _TARGET_ARM_
 
         // On ARM, until proven otherwise by performance numbers, just do the shift.
```

Comment:
This is another weird case where the comment is now talking about the opposite case. I would probably just move this whole comment with the one above.